### PR TITLE
Documentation - library page - fix and enhancement 

### PR DESCRIPTION
**Pull request by rich-newman**
#### PR checklist

- [x] Addresses an existing issue: #2768 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [x] Documentation update

#### Overview of change:

Fix sample code for recent changes, show ES5 JavaScript code that will run with node (#2768)

#### Is there anything you'd like reviewers to focus on?

Not sure if we really need TypeScript AND JavaScript examples

#### CHANGELOG.md entry:

[docs] Fix code on Library Usage page, give JavaScript example

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/tslint](https://github.com/palantir/tslint), @rich-newman! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: docs/usage/library/index.md [hunk 1]
```diff
@@ -18,13 +18,18 @@ yarn add tslint typescript
...
+This code will need to be transpiled to JavaScript to run under Node.js.
+
 ```ts
 import { Linter, Configuration } from "tslint";
 import * as fs from "fs";
 
 const fileName = "Specify input file name";
 const configurationFilename = "Specify configuration file name";
 const options = {
+	fix: "false",
```
Comment (adidahiya) [inline docs/usage/library/index.md]: please use spaces, not tabs, for indentation.

Comment (adidahiya) [inline docs/usage/library/index.md]: also, shouldn't this be a boolean literal, not a string?

File: docs/usage/library/index.md [hunk 2]
```diff
@@ -36,3 +41,27 @@ const configuration = Configuration.findConfiguration(configurationFilename, fil
...
+    formatter: "json",
+    rulesDirectory: "customRules/",
+    formattersDirectory: "customFormatters/"
+};
+var fileContents = fs.readFileSync(fileName, "utf8");
+var linter = new tslint.Linter(options);
+var configuration = tslint.Configuration.findConfiguration(configurationFilename, fileName).results;
+linter.lint(fileName, fileContents, configuration);
+var result = linter.getResult();
+console.log(result);
```
Comment (adidahiya) [inline docs/usage/library/index.md]: let's remove this console.log for consistency with the other code block.

**Pull request merged**